ee/cis/win-10/cis-policy-queries.wip.yml

```diff
@@ -0,0 +1,448 @@
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure 'Remote Desktop Services UserMode Port Redirector (UmRdpService)' is set to 'Disabled'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting determines the redirection of Printers/Drives/Ports for RDP connections.
+  resolution: |
+    Automatic method:
+    Ask your system administrator to establish the recommended configuration via domain GP, set the following UI path to 'Disabled':
+    'Computer Configuration\Policies\Windows Settings\Security Settings\System Services\Remote Desktop Services UserMode Port Redirector'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Services\\UmRdpService\\Start' AND data == 4);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_5.23
```

Comment:
update the `CIS_Level1` to appropriate level throughout document (e.g. this one should be `CIS_Level2`)

Reply:
good catch, updated the levels to match the doc

Reply:
Let me know when you push the updates and I'll approve @marcosd4h

Reply:
@RachelElysia done